examples/nextjs-trpc/src/lib/arcjet.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import arcjet, { slidingWindow } from '@arcjet/next';
+
+export const aj = arcjet({
+	key: process.env.ARCJET_KEY!,
```

Comment:
Why is this using `process.env` instead of the env helper stuff from t3-app?

Reply:
My suggestion seems to work. Commenting above.
```suggestion
	key: env.ARCJET_KEY,
```